discretize/BaseMesh.py

```diff
@@ -377,6 +377,46 @@ def copy(self):
         """
         return properties.copy(self)
 
+    axis_u = properties.Vector3(
+        'Vector orientation of u-direction',
+        default='X',
+        length=1
```

Comment:
`1.0`? not sure if that will be a problem.

Reply:
for the length? should it be an integer or a float? I'm not too familiar with `properties.Vector3`

Reply:
is there a reference for UVW mapping you could add to the docstrings? It may not be a familiar concept to everyone

Reply:
I don’t have any good resources on hand for the UVW axes conventions but I’ll search around and see what I can pull together and put in the docs.